app/views/questions/_form.html.erb

```diff
@@ -1,21 +1,31 @@
 <%= form_tag(response_index_path, method: 'post', class: 'form-check', remote: true, id: 'response-form') do %>
   <div class="modal-body text-left" id='question-container'>
+    <% if alert %>
+      <!-- Alert if problem saving response. -->
+      <p class="alert alert-warning alert-dismissible fade show">
+        <%= alert %>
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </p>
+    <% end %>
     <% if @question.nil? %>
       <h5 class='card-title'>You finished the quiz!</h5>
     <% else %>
       <h5 class='card-title'><%= @question.title %></h5>
       <div id='options-container'>
         <% @question.options.each do |option_title, _| %>
-          <label><%= radio_button_tag(:selected_option, option_title) %> <%= option_title %></label>
```

Comment:
do we want these to be check box?

Reply:
answered in a follow up comment on like 18 of _form.html